pkg/sql/alter_type.go

```diff
@@ -197,6 +195,47 @@ func (p *planner) renameTypeValue(
 	)
 }
 
+func (p *planner) setTypeSchema(params runParams, n *alterTypeNode, schema string) error {
```

Comment:
nit -- usually these functions just take a context, rather than the whole params